src/style/ngm-side-bar.css

```
@@ -70,12 +70,19 @@ ngm-auth:hover {
   position: absolute;
   margin-left: 80px;
   z-index: 1;
+  display: flex;
+  flex-direction: column;
 }
 
```

To have shadow on extension panel:
```suggestion
.ngm-side-bar-panel.ngm-layer-catalog {
z-index: 2;
}
```
Also, need to increase the z-index of the header to keep the shadow on this panel.